lighthouse-core/test/computed/critical-request-chains-test.js

```diff
@@ -12,31 +12,52 @@ const CriticalRequestChains = require('../../computed/critical-request-chains.js
 const NetworkRequest = require('../../lib/network-request.js');
 const NetworkRecords = require('../../computed/network-records.js');
 const ComputedCrc = require('../../computed/critical-request-chains.js');
+const createTestTrace = require('../create-test-trace.js');
+const networkRecordsToDevtoolsLog = require('../network-records-to-devtools-log.js');
 
 const HIGH = 'High';
 const VERY_HIGH = 'VeryHigh';
 const MEDIUM = 'Medium';
 const LOW = 'Low';
 const VERY_LOW = 'VeryLow';
 
-function mockTracingData(prioritiesList, edges) {
+async function createChainsFromMockRecords(prioritiesList, edges, extrasFn) {
   const networkRecords = prioritiesList.map((priority, index) =>
     ({requestId: index.toString(),
-      resourceType: 'fake',
+      url: 'https://www.example.com/' + index,
+      resourceType: index === 0 ? 'Document' : 'Stylesheet',
       frameId: 1,
       finished: true,
       priority,
-      initiatorRequest: null,
+      initiator: null,
       statusCode: 200,
+      startTime: index,
+      responseReceivedTime: index + 0.5,
     }));
 
   // add mock initiator information
   edges.forEach(edge => {
-    const initiator = networkRecords[edge[0]];
-    networkRecords[edge[1]].initiatorRequest = initiator;
+    const initiatorRequest = networkRecords[edge[0]];
+    networkRecords[edge[1]].initiator = {
+      type: 'parser',
+      url: initiatorRequest.url,
+    };
   });
 
-  return networkRecords;
+  if (extrasFn) extrasFn(networkRecords);
+
+  const trace = createTestTrace({topLevelTasks: [{ts: 0}]});
+  const URL = {finalUrl: networkRecords[0].url};
+  const devtoolsLog = networkRecordsToDevtoolsLog(networkRecords, {skipVerification: true});
```

Comment:
which parts weren't verifying? is it something we should catalog to fix in `networkRecordsToDevtoolsLog`?

Reply:
idk something about the initiator. should we fix it?  
![image](https://user-images.githubusercontent.com/4071474/92972623-21cb0f80-f448-11ea-888b-d4b7fc9315a0.png)